stan/math/prim/scal/meta/VectorBuilder.hpp

```diff
@@ -26,19 +26,21 @@ namespace stan {
            typename T4 = double, typename T5 = double, typename T6 = double,
            typename T7 = double>
   class VectorBuilder {
```

Comment:
I made these changes to VectorBuilder so I could compute the return type from data() more easily (this will be the return type for the vectorized rngs)